packages/documentation-site/patternfly-docs/content/design-guidelines/styles/motion/motion.md

```diff
@@ -0,0 +1,154 @@
+1. **Click:**  Shown when a user clicks an element, in order to provide immediate feedback that confirms user actions.
+1. **Loading:** Loading animations, like spinners or progress bars, which keep users informed about the loading process in order to reduce frustration.
+1. **Errors/validation:** Shown when an error occurs during a validation process, in order to draw attention to error messages and guide users as they resolve issues.
+1. **Change:** Smooth transitions between different screens or sections of an application or website that enhance user flow and engagement.
+
+### Anatomy of an animation 
+An animation is composed of 4 elements: triggers, duration, easing, and properties.
+
+1. **Triggers:** The event that causes the animation to happen. These can be: 
+    - User-initiated. For example, a gesture or click. 
+    - System-initiated. For example, after a certain amount of time.
+
+1. **Duration:** Describes how long animations should be. 
+    - Shorter durations are used for simpler effects and smaller animations. For example, fading or color changing.
+    - Longer durations are used for more complex effects and larger-scale animations. For example, page transitions or objects that slide in and out of the screen. Optimal timing can change based on viewport size. 
+
+1. **Easing:** Describes how an element transitions over time.
+
+1. **Properties:** The characteristics of the element being transitioned. Animations may have multiple properties, including: 
+    - Fade in/out
+    - Scale
+    - Rotate
+    - Color transitions
+    - Slide in/out
+    - Expand/collapse
+
+## Developing and designing with motion 
+
+### Figma 
+In Figma, you can find annotations and prototype videos for each animation...
+
+### Motion tokens
+
+PatternFly component animations are created using design tokens that cover the different elements of an animation, including duration, delay, and timing. Like we do with color tokens, we implement motion by using semantic tokens, which are built off of base tokens.
+
+If you aren't familiar with our token system, [refer to this overview](/tokens/about-tokens).
+
+#### Duration 
+Duration tokens specify the length of time that different animation should take to complete.
+
+| Animation | Tokens | Length |
+| --- | --- | --- |
+| Slide-out | -pf--t--global--motion--duration--slide-out--short<br />-pf--t--global--motion--duration--slide-out--default<br />-pf--t--global--motion--duration--slide-out--long | 300<br />400<br />500
+| Slide-in |  |
+| Fade |  |
+| Icon |  |
+<br />
+
+#### Delay 
+Delay tokens specify the length of time that should pass before an animation begins. Delay options include none, short, default, and long.
+
+#### Timing 
+Timing-function tokens specify the easing path that an animation takes. These paths are defined by cubic bezier curves, which define the start and end points ... [fact check]
```

Comment:
Do you want to confirm this before merging this update?

Reply:
yes, ty for pointing out!